common/changes/@microsoft/api-extractor/namespace-import-fix_2022-09-08-14-57.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/api-extractor",
+      "comment": "Fix a recent regression where items exported from both the entry point and from an exported namespace appeared only once in the API doc model (GitHub #3619)",
```

Comment:
@zelliott I updated this change message. Our [change log guidelines](https://rushjs.io/pages/best_practices/change_logs/) recommend to use the word "issue" or "regression" instead of "bug".